cms/djangoapps/contentstore/views/item.py

```diff
@@ -1262,6 +1262,10 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
                 'highlights_preview_only': not COURSE_UPDATE_WAFFLE_FLAG.is_enabled(course.id),
                 'highlights_doc_url': HelpUrlExpert.the_one().url_for_token('content_highlights'),
             })
+        elif xblock.category == 'vertical':
+            xblock_info.update({
+                'icon': xblock.icon,
+            })
```

Comment:
Why is this needed? It seems like there was no icon info here previously.

Reply:
This was needed for Studio (additionally to the theme changes), but as we're extracting this field to a plugin, it sounds fine to remove it now and find an approach suitable for https://github.com/edx/frontend-app-course-authoring.